demosplan/DemosPlanCoreBundle/Repository/StatementRepository.php

```diff
@@ -2032,9 +2031,11 @@ public function copyFileContainers(Statement $originalToCopy, Statement $newOrig
             ->getStatementFileContainers($originalToCopy->getId());
 
         $fileStrings = [];
+        $copiedFileContainers = [];
```

Comment:
this change does not do anything

Reply:
u re right, thx, it's removed here https://github.com/demos-europe/demosplan-core/pull/3431/commits/9642bf72c7b8bd1b666480db653a1bdef637f762